typescript/commands/error_info.py

```diff
@@ -21,7 +21,11 @@ def run(self, text):
             if region.contains(pt):
                 error_text = text
                 break
+
         if len(error_text) > 0:
-            self.view.set_status("typescript_error", error_text)
-        else:
-            self.view.erase_status("typescript_error")
\ No newline at end of file
+            if PHANTOM_SUPPORT:
+                template = '<body><style>div.error {{ background-color: brown; padding: 5px; }}</style><div class="error">{0}</div></body>'
```

Comment:
Does the coloring change at all with the themes? Seeing as you only set background here, and not foreground, could foreground ever default to something in some themes which is unreadable?